openreview/conference/templates/decision_process.py

```diff
@@ -0,0 +1,44 @@
+def process_update(client, note, invitation, existing_note):
+
+    CONFERENCE_ID = ''
+    SHORT_PHRASE = ''
+    AUTHORS_NAME = ''
+    ACCEPTED_AUTHORS_NAME = 'Authors/Accepted'
```

Comment:
this is failing because the function replace matches AUTHOR_NAME here too. I would suggest to use another variable name `ACCEPTED_GROUP_NAME`.